CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security
 
+## 0.1.1
+Released September 28th, 2022
+
+### Fixed 
+- `async tests` - [#5](https://github.com/fivetran/prefect-fivetran/pull/5)
+
```

Comment:
I don't think that this has been released yet, so I recommend a 0.1.0 version. Also, I think we can say which tasks and flow are being released as part of this release.
```suggestion
## 0.1.0
Released September 29th, 2022

### Added 
- `check_fivetran_connector`, `set_fivetran_connector_schedule`, `force_fivetran_connector` and `finish_fivetran_sync` tasks
- `start_fivetran_sync` and `fivetran_sync_flow` flows

```